pymc4/model/base.py

```diff
@@ -83,6 +100,16 @@ def graph(self):
     def observed(self):
         return self._observed
 
+    @property
+    def unobserved(self):
+        unobserved = {}
```

Comment:
this can be initialized as an `OrderedDict`: currently in Python < 3.6, the return value will not be ordered, since you built an (unordered) dictionary, then turned it into an `OrderedDict`.  We're targeting 3.6 and higher though, in which case you do not need the `OrderedDict` at all, since _all_ dicts now maintain insertion order.

That's a long way to say: I would just make a plain dictionary, but if you use `OrderedDict`, it needs to be initialized as such.

Reply:
Nice point